torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -167,6 +172,66 @@ void fuseTransposeIntoGemm(std::shared_ptr<Graph>& graph) {
   }
 }
 
+void pushPackingPastRnn(std::shared_ptr<Graph>& graph) {
+  for (auto it = graph->nodes().begin(); it != graph->nodes().end(); ++it) {
+    auto* n = *it;
+
+    if (n->kind() != kPackPadded) {
+      continue;
+    }
+    if (n->outputs()[0]->uses().size() != 1) {
+      // For now, only handle the case where there is one consumer.
```

Comment:
👍 Multiple consumer case probably could be implemented by "inlining" the PackPadded but we definitely don't have to do it until it's needed.

Reply:
++ yeah I don't want to feature creep this diff any more than necessary